include/rocksdb/utilities/object_registry.h

```diff
@@ -62,10 +79,48 @@ class ObjectLibrary {
     FactoryFunc<T> factory_;
   };  // End class FactoryEntry
  public:
+  // Loads the ObjectLibrary specified by the input value into result
+  // Libraries can be "default", "local", or "dynamic".
```

Comment:
How can a user decide whether a local, or dynamic should be used?

Reply:
If you are loading an external method, it will be a dynamic library.  If you are loading a method from the current executable, it could be put into the local or default library.

Reply:
I'm thinking another name to replace "Local". "local" is not an exclusion case of "dynamic"......